region/norfair/west/Ice Beam Gate Room.json

```diff
@@ -359,6 +359,70 @@
       ],
       "clearsObstacles": ["A", "B", "C"]
     },
+    {
+      "link": [2, 5],
+      "name": "Temporary Blue Chain (Come in Shinecharging)",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 4,
+          "openEnd": 0
+        }
+      },
+      "requires": [
+        "canChainTemporaryBlue"
+      ],
+      "clearsObstacles": ["A", "B"],
+      "note": [
+        "Use a temporary blue chain to jump through the bomb blocks and land on the center of the ledge above.",
```

Comment:
I'd rather not land on the ledge above and go directly for the bomb block.
If you do land above, this needs can4Highmidairmorph.

Reply:
I found going directly for the bomb block to be much more difficult. Just checked with Bizhawk and confirmed that doing it that way requires a frame-perfect morph (at least from the position I tested), whereas from the ledge there is a 2-frame window. Adding can4Highmidairmorph sounds good.